README.md

```diff
@@ -27,7 +27,7 @@ There are some simply convience methods. `get()`, `post()`, `put()`, and `delete
 ```javascript
 import {api} from 'pdjs';
 
-const pd = api({token: 'someToken1234567890'})
+const pd = api({token: 'someToken1234567890' tokenType: 'token'})
```

Comment:
Can you remove this and instead create a specific example for setting a Bearer token?